pkgs/development/libraries/libiconv/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch =
-    lib.optionalString ((stdenv.hostPlatform != stdenv.buildPlatform && stdenv.hostPlatform.libc == "msvcrt") || stdenv.cc.nativeLibc)
+    lib.optionalString ((stdenv.hostPlatform != stdenv.buildPlatform && stdenv.hostPlatform.libc == "msvcrt") || (stdenv.cc != null && stdenv.cc.nativeLibc))
```

Comment:
I'm going to merge without that for now, but we can get back to it tomorrow / in a few hours.